packages/shared/src/hooks/profile/useProfile.ts

```diff
@@ -12,14 +13,11 @@ export function useProfile(initialUser?: PublicProfile): PublicProfile {
     generateQueryKey(RequestKey.Profile, initialUser),
     () => getProfile(initialUser?.id),
     {
-      placeholderData: initialUser,
-      refetchOnReconnect: false,
-      refetchOnMount: false,
-      refetchOnWindowFocus: false,
-      // Make sure the logged in user's profile is always up to date
+      ...disabledRefetch,
+      initialData: initialUser,
       enabled: !!initialUser && isSameUser,
     },
   );
 
-  return user;
+  return user ?? initialUser;
```

Comment:
This might break places where we expect `undefined|null`, just to sanity check since we do have `initialData`

Reply:
I think we can just return `user` since the object would basically return the `initialData`.